web/src/scheduled_messages.js

```diff
@@ -319,3 +323,40 @@ export function update_send_later_options() {
         $("#send_later_options").replaceWith(render_send_later_modal_options(filtered_send_opts));
     }
 }
+
+export function show_schedule_message_indicator() {
+    const scheduled_messages = Object.values(scheduled_messages_data);
+
+    const matching_scheduled_messages = scheduled_messages.filter((scheduled_message) => {
+        if (scheduled_message.failed === true) {
+            return false;
+        }
```

Comment:
@alya product question: Should a scheduled message that failed to send be counted here? I think the correct answer might be yes, in that it is a thing we want you to look at.

(Keep in mind that we do send a DM to users at the moment such a failure occurs)

Reply:
Sure, if they are in the scheduled messages UI, I think it makes sense to count them.

Reply:
I've removed this check.